params/protocol_params.go

```diff
@@ -191,17 +195,47 @@ var (
 	MinBaseFeeInQits              = big.NewInt(5)
 	OneOverBaseFeeControllerAlpha = big.NewInt(100)
 	BaseFeeMultiplier             = big.NewInt(50)
+
+	ConversionLockPeriod uint64 = 2 * BlocksPerWeek
 )
 
 func init() {
-	LockupByteToBlockDepth[0] = ConversionLockPeriod
-	LockupByteToBlockDepth[1] = 30240  // 1.75 days
-	LockupByteToBlockDepth[2] = 60480  // 3.5 days
-	LockupByteToBlockDepth[3] = 120960 // 7 days
-
-	LockupByteToRewardsRatio[1] = big.NewInt(24) // additional 16% WPY
-	LockupByteToRewardsRatio[2] = big.NewInt(10) // additional 20% WPY
-	LockupByteToRewardsRatio[3] = big.NewInt(4)  // additional 25% WPY
+	LockupByteToBlockDepth[0] = ConversionLockPeriod // 2 weeks
+	LockupByteToBlockDepth[1] = 3 * BlocksPerMonth   // 3 months
+	LockupByteToBlockDepth[2] = 6 * BlocksPerMonth   // 6 months
+	LockupByteToBlockDepth[3] = BlocksPerYear        // 12 months
+
+	// These lockup bytes are divided by 10,000 while calculating the reward
+	// The first value represents the interest rate for the first year, the second
+	// value represents the terminal rate
+	LockupByteToRewardsMultiple[1] = []uint64{3500, 218}
+	LockupByteToRewardsMultiple[2] = []uint64{10000, 625}
+	LockupByteToRewardsMultiple[3] = []uint64{25000, 1562}
```

Comment:
What's the rationale for 250% interest?

Reply:
changed this math again for not losing precision, directly multiplying by a value then dividing by 10,000